web/tests/stream_data.test.cjs

```diff
@@ -1937,6 +1937,10 @@ test("can_access_stream_email", ({override}) => {
         name: "social",
         stream_id: 2,
         is_archived: false,
+        is_web_public: false,
+        subscribed: false,
+        invite_only: false,
+        history_public_to_subscribers: false,
```

Comment:
This configuration is not possible practically in non-zephyr realms, public streams always have history public to subscribers. It is good to have test data similar to what we have in production.